### PR TITLE
Fix Windows support for building documentation

### PR DESCRIPTION
**Pull request by kibertoad**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +35,7 @@
     "highlight.js": "^9.5.0",
     "http-proxy": "^1.14.0",
     "ip": "^1.1.3",
-    "knex": "*",
+    "knex": "^0.16.3",
```
Comment (elhigu) [inline package.json]: This should be left to * to get always the latest version for bundling knex and getting the changelog...

File: scripts/build.js [hunk 2]
```diff
@@ -58,7 +58,11 @@ const env = {
...
 
-runCmd('docs-server', 'node_modules/.bin/babel-node scripts/server.js', { env }).then(() => {
+const isWin = process.platform === "win32"
+const prefix = isWin ? 'call ' : ''
+const extension = isWin ? '.cmd' : ''
+
+runCmd('docs-server', `${prefix}node_modules/.bin/babel-node${extension} scripts/server.js`, { env }).then(() => {
   console.log('Building webpack for production'.cyan)
-  return runCmd('webpack', `node_modules/.bin/webpack -p --config scripts/webpack.config.js`, { env })
+  return runCmd('webpack', `${prefix}node_modules/.bin/webpack${extension} -p --config scripts/webpack.config.js`, { env })
```
Comment (elhigu) [inline scripts/build.js]: 👍

File: package.json (hunk 1 above)
Comment (kibertoad) [reply package.json]: Don't remember why I changed it, maybe it was accidental. Changed back.

**Comment by kibertoad** [comment]
@elhigu Ouch, package-lock was definitely an accident. Removed.

File: package.json (hunk 1 above)
Comment (elhigu) [reply package.json]: Maybe some VSCode or npm stuff does that automatically... I remember having to fix that  every now and then when I've been updating docs.

Comment (kibertoad) [reply package.json]: Could be :)

**Comment by elhigu** [comment]
And probably we should also add that package-lock to .gitignore file to prevent it from being included in PRs. Looks like it already had slipped in by accident.

**Comment by kibertoad** [comment]
@elhigu Surprisingly enough, it IS in gitignore :D

**Comment by elhigu** [comment]
😃🤦‍♂️

**Pull request merged**